tests/drivers/build_all/sensor/prj.conf

```diff
@@ -35,6 +35,7 @@ CONFIG_ENS210=y
 CONFIG_FDC2X1X=y
 CONFIG_FXAS21002=y
 CONFIG_FXOS8700=y
+CONFIG_HIDS=y
```

Comment:
With the Kconfig changes, you don't need to add `CONFIG_HIDS` here.

Reply:
Done.